Source/src/resources/ResourceMesh.h

```diff
@@ -6,23 +6,38 @@ namespace Hachiko
     class ResourceMesh : public Resource
     {
     public:
+        struct Bone
+        {
+            char name[100]; // TODO: find a better size
```

Comment:
You can use ```PATH_MAX``` which is the maximum allowed file name length on windows. (256)

Reply:
nice ❤️